app/views/course/assessment/question/voice_responses/edit.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+question = @voice_response_question
+question_assessment = @question_assessment
+
+json.partial! 'form', locals: {
+  question: question,
+  question_assessment: question_assessment,
+  new_question: false,
+  course: current_course
+}
+
+json.question do
+  json.partial! 'course/assessment/question/form', question: question
+  json.skillIds question_assessment.skill_ids
```

Comment:
You can move `skillIds` to `'course/assessment/question/form'` too. Dont forget to remove the same line in `app/views/course/assessment/question/multiple_responses/edit.json.jbuilder` after `json.skillIds` has been abstracted